content/master/concepts/usages.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Usages
+weight: 95
+state: alpha
+alphaVersion: "1.14"
+description: "Usage defines a usage relationship for Managed Resources or Composites"
+---
+
+A `Usage` is a Crossplane resource that defines a usage relationship for a 
+Managed Resource or a Composite Resource. Two main use cases for the Usages are
+as follows:
+
+1. Protecting a resource from accidental deletion.
+2. Deletion ordering by ensuring that a resource isn't deleted before the 
+   deletion of its dependent resources.
+
+See the [Usage for Deletion Protection](#usage-for-deletion-protection) for the
```

Comment:
I would add "section" after the two links so it reads "See the foo section for the first use case and the bar section for the second one."